cmd/tailscaled/tailscaled_ios.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build go1.18
+// +build go1.18
+
+package main // import "tailscale.com/cmd/tailscaled"
+
+import (
+	"os"
+	"runtime/debug"
+)
+
+func init() {
+	// We aren't very performance sensitive, and the parts that are
+	// performance sensitive (wireguard) try hard not to do any memory
+	// allocations. So let's be aggressive about garbage collection,
+	// unless the user specifically overrides it in the usual way.
+
+	// TODO(raggi): detect ios 15+ and do not apply this there. This tuning
+	// creates a strong throughput reduction in userspace-networking mode as it
+	// clears the sync.Pools too frequently.
+	if _, ok := os.LookupEnv("GOGC"); !ok {
+		debug.SetGCPercent(10)
+	}
```

Comment:
There is no tailscaled on iOS and even if there were, there'd be no way for users to set environment variables anyway.